client/src/components/ExerciseCard.js

```diff
@@ -0,0 +1,26 @@
+import { useParams } from 'react-router';
+
+export default function ExerciseCard({ exercises }) {
+  const { id } = useParams();
+
+  const findExercise = exercises.find((exercise) => {
+    return exercise._id === id;
+  });
+
+  return (
+    <div>
+      <h1>Exercise CARD</h1>
+      <p>{findExercise.titel}</p>
+      <p>{findExercise.description}</p>
+      {findExercise.imageURL.map((img) => {
+        return <img src={img} />;
+      })}
```

Comment:
I was confused by mapping over a singular name `imageURL`, maybe rename to `imageURLs` to indicate an array. 

*Always* assign a `key` when mapping.

If you don't need an alt attribute maybe explicitly set it to an empty string to indicate that the image holds no additional information?